dir/path.go

```diff
@@ -120,3 +153,18 @@ func (p *PathManager) CachedSignatureStoreDirPath() string {
 	checkError(err)
 	return path
 }
+
+func getPathForWrite(user UnionDirFS, system UnionDirFS, item ...string) string {
+	var (
+		path string
+		err  error
+	)
+	if WriteLevel == SystemLevel {
+		path, err = system.GetPath(item...)
+	} else {
+		path, err = user.GetPath(item...)
+	}
```

Comment:
When system == p.SystemConfigFS and user == p.UserConfigFS, then this logic does not cover the case of writing/updating/removing from both levels. Therefore, to work on both system and user levels (e.g. add a file to both levels), one needs to explicitly change the WriteLevel var outside this func by calling 'WriteLevel = 0', and the caller has to remember the current WriteLevel value to avoid miss operations on the wrong level.

So it looks to me that WriteLevel should be an argument of getPathForWrite. Functions calling getPathForWrite shall pass in WriteLevel, which is decided by the caller.

Reply:
Thanks. Added writeLevel argument.